framework/src/modules/http_api/controllers/voters.js

```diff
@@ -123,7 +121,7 @@ VotersController.getVoters = async function(context, next) {
 		data.username = data.username || '';
 
 		const voters = await storage.entities.Account.get(
-			{ votedDelegatesPublicKeys_in: [delegate.publicKey] },
+			{ votedDelegatesPublicKeys_in: [`"${delegate.publicKey}"`] }, // Need to add quotes for PSQL array search
```

Comment:
Same here

Reply:
Changed to `votedDelegatesPublicKeys` accepting a single PK. Quotes need to be added anyway but the query it's simpler now.

Reply:
If the above works, use `'mem_accounts."votedDelegatesPublicKeys" @> ${votedDelegatesPublicKeys:name}'` here

Reply:
If the above works, use `'mem_accounts."votedDelegatesPublicKeys" @> ${votedDelegatesPublicKeys:name}'` here without the quotes as well

Reply:
Does not work please check previous answer @lsilvs